### ISSUE TITLE
bot doesnt unfollow anyone 

### ISSUE THREAD

**Issue by Menawan** [instabot-py/instabot.py#2248]
and ı dont get ant log about ıt it is like he doesnt see the line but when ı change the codes ın the ınstabot ı see that ıt doesnt return the 200 code for unfollow can you guys help me

**Comment by Menawan** [comment]
ıt is make unfollow with "pk" not with username anymore write pk ınstead of username..
you can close the entry..

**Comment by eduardoegos** [comment]
a lot of people are complaining about this issue. did you solve it? how? where is this pk thing?

**Comment by yurilaaziz** [comment]
I am not sure to understand what you're saying. what 'pk' stand for?

**Comment by smoki3** [comment]
Here also... the bot does not unfollow anyone. even if place the follow_per_day to 0 and unfollow_per_day to 300

**Comment by smoki3** [comment]
I think this is the problem:

https://github.com/instabot-py/instabot.py/issues/2253

If you use it for more than one account then it will go crazy

**Comment by smoki3** [comment]
`2019-06-28 18:36:46,487 - InstaBot - DEBUG - Trying to unfollow #1:
2019-06-28 18:36:46,495 - InstaBot - DEBUG - Getting user info : the_happy_princee
2019-06-28 18:36:47,165 - InstaBot - DEBUG - Checking user info..
2019-06-28 18:36:47,166 - InstaBot - DEBUG - Follower : 388
2019-06-28 18:36:47,166 - InstaBot - DEBUG - Following : 745
2019-06-28 18:36:47,166 - InstaBot - DEBUG - Media : 86
2019-06-28 18:36:47,166 - InstaBot - DEBUG -    >This is a normal account
2019-06-28 18:36:47,167 - InstaBot - DEBUG -    >This user is active
2019-06-28 18:36:47,167 - InstaBot - DEBUG -    >This account is following you
2019-06-28 18:36:47,167 - InstaBot - DEBUG -    >You are following this account
2019-06-28 18:36:47,167 - InstaBot - DEBUG - current_user :the_happy_princee
2019-06-28 18:36:47,670 - InstaBot - INFO - Unfollowed: https://www.instagram.com/the_happy_princee/ #1.
2019-06-28 18:37:35,819 - InstaBot - DEBUG - Get Media by tag: offroading
2019-06-28 18:38:45,592 - InstaBot - DEBUG - Get Media by tag: offroading
2019-06-28 18:39:53,374 - InstaBot - DEBUG - Get Media by tag: vwhome
2019-06-28 18:41:04,685 - InstaBot - DEBUG - Get Media by tag: caroftheday
2019-06-28 18:42:12,805 - InstaBot - DEBUG - Get Media by tag: vwhome
2019-06-28 18:42:41,789 - InstaBot - DEBUG - osram.automotive : Get media id on recent feed
2019-06-28 18:42:43,012 - InstaBot - DEBUG - Media in recent feed = 12
2019-06-28 18:42:43,174 - InstaBot - DEBUG - Trying to unfollow #2:
2019-06-28 18:42:43,181 - InstaBot - DEBUG - Getting user info : the_happy_princee
2019-06-28 18:42:43,820 - InstaBot - DEBUG - Checking user info..
2019-06-28 18:42:43,821 - InstaBot - DEBUG - Follower : 387
2019-06-28 18:42:43,822 - InstaBot - DEBUG - Following : 745
2019-06-28 18:42:43,822 - InstaBot - DEBUG - Media : 86
2019-06-28 18:42:43,823 - InstaBot - DEBUG -    >This is a normal account
2019-06-28 18:42:43,823 - InstaBot - DEBUG -    >This user is active
2019-06-28 18:42:43,824 - InstaBot - DEBUG -    >This account is following you
2019-06-28 18:42:43,824 - InstaBot - DEBUG -    >You are NOT following this account
2019-06-28 18:42:43,825 - InstaBot - DEBUG - current_user :the_happy_princee
2019-06-28 18:42:44,320 - InstaBot - INFO - Unfollowed: https://www.instagram.com/the_happy_princee/ #2.
`


and it looks like it does not delete the entries in the db. It is trying to unfollow the same account twice. first time it works secound time we are not following them

**Comment by x3l51** [comment]
The problem is still there, even with the latest instabot version.

```
2019-07-09 21:26:54,812 - InstaBot - DEBUG - Trying to unfollow #1:
2019-07-09 21:26:54,814 - InstaBot - DEBUG - Looks like there is nobody to unfollow.
```
Yet, it should unfollow around 1200 profiles.

**Comment by nickgryg** [comment]
This issue is fixed in recent instabot-py versions after 0.5.13. If you want to use it, just perform:
```
python3 -m pip install instabot-py
```
